src/hip_module.cpp

```diff
@@ -452,10 +452,7 @@ hipError_t ihipModuleLaunchKernel(hipFunction_t f,
 
         lp.av->dispatch_hsa_kernel(&aql, config[1] /* kernarg*/, kernArgSize, 
                                   (startEvent || stopEvent) ? &cf : nullptr
-#define USE_NAMED_KERNEL 0
-#if USE_NAMED_KERNEL
                                   , f->_name.c_str()
```

Comment:
Yep, looks like this was post 1.6.  Here's an updated version with WW check - I know we had it by 17300 so was conservative.